NETDefault/Gateway.Buckaroo/HmacDelegatingHandler.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Net.Http.Headers;
+using System.Security.Cryptography;
+using System.Text;
+using Microsoft.Extensions.Options;
+
+namespace Gateway.Buckaroo;
+
+public class HmacDelegatingHandler : DelegatingHandler
+{
+    private readonly BuckarooConfiguration _configuration;
+
+    public HmacDelegatingHandler(IOptionsSnapshot<BuckarooConfiguration> configuration)
+    {
+        _configuration = configuration.Value;
+    }
+
+    protected async override Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+    {
+        var requestContentBase64String = string.Empty;
+        var requestUri = Uri.EscapeDataString(request.RequestUri!.Authority + request.RequestUri.PathAndQuery).ToLower();
+
+        var epochStart = new DateTime(1970, 01, 01, 0, 0, 0, 0, DateTimeKind.Utc);

```

Comment:
`DateTimeOffset` can help with unix timestamp stuff.

Reply:
Elaborate please

Reply:
Instead of using `DateTime`, you can use `DateTimeOffset` which has these unix epoch stuff in it already: https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.tounixtimeseconds?view=net-6.0#system-datetimeoffset-tounixtimeseconds